app/views/posts/show.html.erb

```diff
@@ -1 +1,12 @@
-<h1>Here should be some more details about a specific post of a certain user</h1>
\ No newline at end of file
+<div class="post">
+  <div class="post-info">
+    <h3>Post by <%= @post.author.name %></h3>
+    <p>Commets: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></p>
+  </div>
+  <p class="post-body"><%= @post.text %></p>
+  <div class="comments">
+    <% @post.comments.each do |comment| %>
+      <p><%= comment.author.name %>: <%= comment.text %></p>
+    <% end %>
+  </div>
+</div>
```

Comment:
- [ ] Again here, kindly follow the given design, this includes the border, positioning and layout of components as this is a requirement

Given wireframe
![image](https://user-images.githubusercontent.com/81932478/214706543-f0dfcf07-15d2-4ff6-a7f6-cfda834bac3d.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/214706676-34265e53-8c30-40b0-8834-906d9f3c87da.png)